frontend/src/modules/entitieslist/components/EntitiesList.js

```diff
@@ -1,5 +1,6 @@
 /* @flow */
 
+import type { Node, Element } from 'React';
 import React from 'react';
```

Comment:
I'd prefer to do this the other way around, use `React` as type namespace, and explicitly import `createRef`.